### PR TITLE
docs(scheme): add custom scheme creation docs

### PR DESCRIPTION
**Pull request by JoaoPedroAS51**

### CODE REVIEW

File: docs/guide/scheme.md [hunk 1]
```diff
@@ -20,7 +20,84 @@ auth: {
...
+Sometimes the included schemes doesn't match your requirements. Creating your own scheme will provide flexibility you need. You can create a new scheme from scratch or extend an existing scheme.
+
+> **Note** A list of available scheme methods will be added in future releases.
+
+Before creating custom schemes, you will need to add `@nuxtjs/auth` to `transpile`.
+
+`nuxt.config.js`
+```js
+build: {
+  transpile: ['@nuxtjs/auth']
```
Comment (pi0) [inline docs/guide/scheme.md:38]: This should be already done by auth module to self-transpile:https://github.com/nuxt-community/auth-module/blob/dev/src/module/index.ts#L41

Comment (JoaoPedroAS51) [reply docs/guide/scheme.md:38]: The docs are for `master` branch, so it doesn't self-transpile :)

Comment (JoaoPedroAS51) [reply docs/guide/scheme.md:38]: I'm improving more docs for custom scheme, but after merge to `dev` branch, I'll remove the transpile part.

Comment (pi0) [reply docs/guide/scheme.md:38]: Thanks :) BTW for master, providers are copied to `.nuxt` (which should be imported using relative path not NPM package)

Comment (JoaoPedroAS51) [reply docs/guide/scheme.md:38]: Hmm ok. I'll fix that.

**Pull request merged**